spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/sql/TestSparkBucketFunction.java

```diff
@@ -74,10 +74,14 @@ public void testSpecValues() {
         .as("Spec example: hash(2017-11-16T22:31:08) = -2047944441")
         .isEqualTo(-2047944441);
 
-    assertThat(new BucketFunction.BucketString().hash("iceberg"))
+    assertThat(new BucketFunction.BucketString().hash("iceberg".getBytes(StandardCharsets.UTF_8)))
```

Comment:
Nit: Same as above, we could just leave this assertion the same as earlier just to keep the diff smaller.